skrf/network.py

```diff
@@ -2282,7 +2282,14 @@ def get_buffer() -> io.StringIO:
                 output.write('!Data is not renormalized\n')
                 output.write('# {} S {} R\n'.format(ntwk.frequency.unit, form))
             else:
-                output.write('# {} S {} R {} \n'.format(ntwk.frequency.unit, form, r_ref))
+                # Write "r_ref.real" instead of "r_ref", so we get a real number "a" instead
+                # of a complex number "(a+0j)", which is unsupported by the standard Touchstone
+                # format (non-HFSS). We already checked in the beginning that "r_ref" must be
+                # real in this case (write_z0 == False).
+                assert r_ref.imag == 0, "Complex reference impedance is encountered when " \
```

Comment:
`assert` raises an `AssertionError`, which breaks the program, correct?
I think a `RuntimeWarning` would be better, see this example:
https://github.com/scikit-rf/scikit-rf/blob/21b3409966676d28ac5dfc29552b0eff6c408750/skrf/network.py#L560-L562